test/fixed_rank.jl

```diff
@@ -166,6 +166,20 @@ include("utils.jl")
                 @test oneP == one(x)
                 oneV = UMVTVector(one(zeros(3, 3)), one(zeros(2, 2)), one(zeros(2, 2)), 2)
                 @test oneV == one(v)
+
+                # broadcasting
+                @test axes(w) === ()
+                wc = copy(w)
+                @test wc.U !== w.U
```

Comment:
Why is this an `!==` test? I would have thought copying `U` should make the `U`s the same?

Reply:
They should be equal but represented using different data. For mutable collections we can't have `copy(x)=x` and this tests for it.
```julia
julia> a = randn(3,3);

julia> b = copy(a);

julia> a==b
true

julia> a===b
false
```

Reply:
Oh, and I misread `!==` because it is the negative form of `===` right? I thought it was `!(==)` but that's `!=`.
But the values should be the same, so `wc.U == w.U` should be true, can we check for that, too?

Reply:
Yes, exactly -- `!==` is the negation of `===` while `!=` is the negation of `==`. I'll add the test for `==` and a comment.

Reply:
Then this was my confusion to begin with ;) great to also have a `==` test, with that I can accept this PR :)